api/server/router/container/container.go

```diff
@@ -60,6 +60,7 @@ func (r *containerRouter) initRoutes() {
 		// PUT
 		local.NewPutRoute("/containers/{name:.*}/archive", r.putContainersArchive),
 		// DELETE
+		local.NewDeleteRoute("/containers/_all", r.deleteAllContainers),
```

Comment:
From what I recall, the theory says it should be `containers/` with the trailing slash, to refer to the collection, analogous to how on a file system a trailing `/` refers to a folder and no trailing `/` refers to a file. I wish I had a reference, but I do not.

That said, it would be nice to have a specific document or reference to consult.

I can see how it could seem dangerous to a user to specify it that way, in case a client lib fails to notice a null container and prints an empty string.